README.md

```diff
@@ -42,6 +42,33 @@ pip install -U pip setuptools wheel
 pip install cymem
 ```
 
+### Free threading
+
+`cymem` has support for being built and run under free-threaded CPython.
+Currently `Pool` is not thread safe when used from multiple threads at once;
+**please avoid sharing a single** `Pool` instance between threads.
+
+Also remember to declare `freethreading_compatible` for the example case below:
```

Comment:
This isn't `cymem`-specific, it's general advice for Cython usage with free-threading. I'd remove this sentence and the code example below.